tests/unit/test_logging_config.py

```
@@ -0,0 +1,145 @@
+        assert "password" not in logger.name.lower()
+        assert "secret" not in logger.name.lower()
+        assert "token" not in logger.name.lower()
+
+    def test_logger_functional_for_normal_messages(self):
+     """Test that logger works normally without raising exceptions."""
+    logger = setup_logging("test_sensitive")
+    try:
+        logger.info("User logged in successfully")
+        logger.warning("Test warning")
+        logger.error("Test error")
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)